_docs-2-0/administration/monitoring-metrics.md

```diff
@@ -95,8 +95,13 @@ from growing without bound.
 Each server process should have log messages from the Metrics2 library about the sinks that were created. Be sure to check
 the Accumulo processes log files when debugging missing metrics output.
 
-For additional information on configuring Metrics2, visit the [Javadoc page for Metrics2](https://hadoop.apache.org/docs/current/api/org/apache/hadoop/metrics2/package-summary.html).
+Below are additional resources for configuring Metrics2: 
 
+* [Javadoc page for Metrics2][metrics2-javadoc]
+* [Blog post][grafana-post] about configuring Grafana/InfluxDB to view Accumulo metrics2
+
+[grafana-post]: /blog/2018/03/22/view-metrics-in-grafana/
```

Comment:
Should this link include the normal relative path prefix? `{{ site.baseurl }}`/blog/...`

Reply:
sure i can do that